README.md

```
@@ -11,14 +11,19 @@ Configuration files that enhance Cursor AI editor experience with custom rules a
 [Cursor AI](https://cursor.sh/) is an AI-powered code editor. `.cursorrules` files define custom rules for Cursor AI to follow when generating code, allowing you to tailor its behavior to your specific needs and preferences.
 
 ## Sponsorships
-<p align="left">
+<p align="center">
+    <h3><a href="https://go.warp.dev/awesome-cursorrules">Warp - Built for coding with multiple AI agents</h3>
+	  <a href="https://go.warp.dev/awesome-cursorrules">
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)